hardware/BleBox.h

```diff
@@ -44,6 +44,10 @@ class BleBox : public CDomoticzHardwareBase
 	boost::shared_ptr<boost::thread> m_thread;
 	std::map<const std::string, const int> m_devices;
 	boost::mutex m_mutex;
+	
+	float LimitlessRGBWcHueState;
```

Comment:
Could you describe what you 'fixed' ?
Member functions should start with m_

Reply:
"Limitless" support did not work at all(only "LedOn" worked), I rewritten and tested it on my wLightBox device.

I did not create new functions, it's just variables to remember the state in the session.